src/main/java/io/github/syst3ms/skriptparser/parsing/SyntaxParser.java

```diff
@@ -63,6 +64,12 @@
     // Gradle requires the cast, but IntelliJ considers it redundant
     public static final PatternType<Object> OBJECTS_PATTERN_TYPE = new PatternType<>((Type<Object>) TypeManager.getByClass(Object.class).orElseThrow(AssertionError::new), false);
 
+    public static final ExpressionInfo<ExprBooleanOperators, Boolean> EXPRESSION_BOOLEANOPERATORS
```

Comment:
Right, I forgot to mention it, this should rather be EXPRESSION_BOOLEAN_OPERATORS, not separating the words with an underscore is against the usual conventions for SCREAMING_SNAKE_CASE.